static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -364,7 +382,9 @@ define([
                 });
             }
         });
-
+        if($('.heroic--video')){
```

Comment:
You need to check for `$('.heroic--video').length > 0` here as `$` always returns an array (even if it's empty) so will always be true.